xla/service/cpu/onednn_util.h

```diff
@@ -17,25 +17,35 @@ limitations under the License.
 #define XLA_SERVICE_CPU_ONEDNN_UTIL_H_
 #if defined(INTEL_MKL) && defined(ENABLE_ONEDNN_V3)
 
-#include "xla/xla_data.pb.h"
 #include "tsl/platform/cpu_info.h"
+#include "xla/xla_data.pb.h"
 
 namespace xla {
 namespace cpu {
 
 inline bool IsSupportedType(xla::PrimitiveType dtype) {
   using tsl::port::CPUFeature;
-  static bool is_bf16_supported = TestCPUFeature(CPUFeature::AVX512_BF16) ||
-                                  TestCPUFeature(CPUFeature::AMX_BF16);
+  bool is_supported = false;
```

Comment:
Nit: We can just get rid of this variable and just return the value/expression directly, e.g., `return true`, `return TestCPUFeature...`.

Reply:
Thanks! Corrected.